pkg/db-repo/repository.go

```diff
@@ -217,20 +217,33 @@ func (r *repository) Delete(ctx context.Context, value ModelBased) error {
 	return err
 }
 
+func (r *repository) isQueryableModel(db *gorm.DB, model interface{}) bool {
+	tableName := db.NewScope(model).TableName()
```

Comment:
why don't you use the orm on the repository?
	tableName := r.orm.NewScope(model).TableName()

Reply:
Applied